indico/util/mdx_latex.py

```diff
@@ -253,33 +268,37 @@ def tolatex(self, ournode):
         elif ournode.tag == 'q':
             buffer += "`%s'" % subcontent.strip()
         elif ournode.tag == 'p':
-            buffer += '\n%s\n' % subcontent.strip()
+            # do not escape if it is latex already
+            if contains_supported_latex_symbols(subcontent):
```

Comment:
Why would an abstract contain LaTeX code? We only handle LaTeX math delimited by `$...$` or `$$...$$`.